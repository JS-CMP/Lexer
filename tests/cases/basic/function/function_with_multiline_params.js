function test(a, b, c, d) {
    return a + b + c + d
}



test(
    1,
    2,
    3,
    4
)