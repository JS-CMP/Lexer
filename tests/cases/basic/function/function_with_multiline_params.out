#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {JS::Any test;test = JS::Any(std::make_shared<JS::Function>([&](const JS::Any &thisArg, const JS::Any &arguments) -> JS::Any {JS::Any a = arguments["0"];JS::Any b = arguments["1"];JS::Any c = arguments["2"];JS::Any d = arguments["3"];return a+b+c+d;return JS::Any();}));test(JS::Any(1),JS::Any(2),JS::Any(3),JS::Any(4));}
